x-pack/plugins/ml/public/jobs/components/custom_url_editor/list.js

```diff
@@ -121,18 +133,31 @@ export class CustomUrlList extends Component {
       const otherUrls = [...customUrls];
       otherUrls.splice(index, 1);   // Don't compare label with itself.
       const isInvalidLabel = !isValidLabel(label, otherUrls);
-      const invalidLabelError = (isInvalidLabel === true) ? ['A unique label must be supplied'] : [];
+      const invalidLabelError = (isInvalidLabel === true) ? [
+        intl.formatMessage({
+          id: 'xpack.ml.customUrlEditorList.labelIsNotUniqueErrorMessage',
+          defaultMessage: 'A unique label must be supplied'
+        })
+      ] : [];
 
       // Validate the time range.
       const timeRange = customUrl.time_range;
       const isInvalidTimeRange = !(isValidTimeRange(timeRange));
-      const invalidIntervalError = (isInvalidTimeRange === true) ? ['Invalid format'] : [];
+      const invalidIntervalError = (isInvalidTimeRange === true) ? [
+        intl.formatMessage({
+          id: 'xpack.ml.customUrlEditorList.invalidFormatErrorMessage',
```

Comment:
`invalidTimeRangeFormatErrorMessage` rather than ``nvalidFormatErrorMessage` would be a better label I think